.github/workflows/ci-windows.yml

```diff
@@ -140,6 +140,7 @@ jobs:
         build\ext\libpng\libpng16_static.lib
         build\ext\zlib\zlibstatic.lib
         build\avif.lib
+        ws2_32.lib ntdll.lib userenv.lib bcrypt.lib advapi32.lib
```

Comment:
On the main branch, currently, system libraries are being included in the merged archive (in merge_static_libs.cmake). Following feedback in past pull requests, I've made it so that only local libraries are included in the merged archive. And so now these have to be explicitly linked against alongside avif.lib. This will be less burdensome once we've updated libavif.pc to document these dependencies, so that users can use the output of `pkg-config --static --libs libavif`. But in the spirit of keeping this pull request more focused I didn't include any pkg-config changes.